tests/python/unittest/test_meta_schedule_schedule_rule_mlt_tc.py

```diff
@@ -988,15 +988,15 @@ def conv2d_1x1_0(inputs: T.Buffer((1, 16, 16, 64), "float16"), weight: T.Buffer(
                                 v2, v3 = T.axis.remap("SS", [ax2_1, ax3])
                                 v4_o = T.axis.spatial(1, 0)
                                 v5_o = T.axis.spatial(1, 0)
-                                T.reads(conv2d_nhwc_reindex_shared_wmma_accumulator[v0, v1, v2, v3, 0:16, 0:16])
-                                T.writes(conv2d_nhwc_reindex_shared[v0, v1, v2, v3, 0:16, 0:16])
+                                T.reads(conv2d_nhwc_reindex_shared_wmma_accumulator[v0, v1, 0, 0, 0:16, 0:16])
```

Comment:
I think this kind of chengs may break current tensorcore support IMO. cc @junrushao @vinx13

Reply:
yes these unit iters can be important in some workloads (1x1 conv, or batch size = 1 gemm)

Reply:
Thanks for the reminder. Do you mean tensorcore needs read/write ranges to be in a specific form? What kind of requirement does it like?

Reply:
it doesn't require specific forms of reads/writes. however, it needs iter vars with extent = 1 to be kept for dimensions used for padding

Reply:
Thanks. Do you mean preserving the for loop with unit extend that `preserve_unit_iters` ensure? The for loop in this case is not changed, only the read/write range is changed here. Does it require all the iter vars be kept anywhere in reads/writes and expressions even if it's always constant 0?

Reply:
I verified these workloads and confirmed this PR doesn't break auto tensorization. Keeping these unit iters preserve more semantics of original IR, but I don't have strong opinion